nexuse2e-core/src/test/java/org/nexuse2e/util/DateTimeTest.java

```diff
@@ -26,105 +28,111 @@
     @Before
     public void setUp() throws Exception {
     }
-    
+
     @Test
     public void constructFromShortArray() {
         short[] parameters = {19, 70, 1, 1, 1, 0, 3, 0};
         DateTime dt = new DateTime(parameters);
-        
-        assertTrue(19 == dt.getCentury());
-        assertTrue(70 == dt.getYear());
-        assertTrue(1 == dt.getMonth());
-        assertTrue(1 == dt.getDay());
-        assertTrue(1 == dt.getHour());
-        assertTrue(0 == dt.getMinute());
-        assertTrue(3 == dt.getSeconds());
-        assertTrue(0 == dt.getMilli());
+
+        assertEquals(19, dt.getCentury());
+        assertEquals(70, dt.getYear());
+        assertEquals(1, dt.getMonth());
+        assertEquals(1, dt.getDay());
+        assertEquals(1, dt.getHour());
+        assertEquals(0, dt.getMinute());
+        assertEquals(3, dt.getSeconds());
+        assertEquals(0, dt.getMilli());
     }
-    
-    @Test(expected=IllegalArgumentException.class)
+
+    @Test(expected = IllegalArgumentException.class)
     public void constructFromShortArrayWithInvalidLength() {
         short[] parameters = {19, 70, 1, 1};
         DateTime dt = new DateTime(parameters);
     }
 
     @Test
     public void constructFromLong() {
+
+        int offset = TimeZone.getDefault().getRawOffset();
+
         DateTime dt = new DateTime(3600L);
-        
-        assertTrue(19 == dt.getCentury());
-        assertTrue(70 == dt.getYear());
-        assertTrue(1 == dt.getMonth());
-        assertTrue(1 == dt.getDay());
-        assertTrue(1 == dt.getHour());
-        assertTrue(0 == dt.getMinute());
-        assertTrue(3 == dt.getSeconds());
-        assertTrue(600 == dt.getMilli());
+
+        assertEquals(19, dt.getCentury());
+        assertEquals(70, dt.getYear());
+        assertEquals(1, dt.getMonth());
+        assertEquals(1, dt.getDay());
+        assertEquals(0 + (offset / (60 * 60 * 1000)), dt.getHour());
```

Comment:
This calculation is somewhat hard to understand. I think it should be explained.

Also there are timezones with offsets that are not a multiple of a full hour. For example the
Afghanistan Time (AFT) has an offset of +4:30h or neapel has +5:41:16.

I think we should keep the expected values non-dynamic since its pretty easy to screw up this kind of calculations. It also makes the test somewhat obtuse.

Can we set the timezone to a known fixed value for this test? Maybe there is an annotation for junit(We should make sure that parralell threads remain uneffected).

Or maybe we can set the TimeZone in the DateTime object.